src/desktop/apps/auth2/routes.js

```diff
@@ -0,0 +1,45 @@
+import { renderLayout } from '@artsy/stitch'
+import { AuthStatic } from './components/AuthStatic.tsx'
+
+export const index = async (req, res, next) => {
+  let mode
+  switch (req.path) {
+    case '/login':
+      mode = 'login'
+      break
+    case '/signup':
+      mode = 'register'
+      break
+    case '/reset_password':
+      mode = 'reset_password'
+      break
+    default:
+      mode = 'login'
+      break
+  }
+
+  try {
+    const layout = await renderLayout({
+      basePath: __dirname,
+      layout: '../../components/main_layout/templates/react_blank_index.jade',
+      config: {
+        styledComponents: true,
+      },
+      blocks: {
+        head: 'templates/meta.jade',
```

Comment:
Sorry for always pinging on line of code, but can this be replaced with a js file and a `<Fragment />`? The less jade we use in new code the better, and with React's Fragment support using `.jade` files in the head is no longer necessary.

Reply:
no need to apologize. the gentle nudge is the only way it's gonna stick. i'll address it in the next commit.